tests/osaka/eip7692_eof_v1/eip4750_functions/test_callf_execution.py

```diff
@@ -663,3 +668,44 @@ def test_callf_max_stack(
         sender=sender,
     )
     state_test(env=env, pre=pre, post=post, tx=tx)
+
+
+def test_callf_retf_memory_context(
+    state_test: StateTestFiller,
+    pre: Alloc,
+):
+    """Verifies CALLF and RETF don't corrupt memory."""
+    env = Environment()
+    storage = Storage()
+    contract_address = pre.deploy_contract(
+        code=Container(
+            sections=[
+                Section.Code(
+                    Op.SSTORE(storage.store_next(value_code_worked), value_code_worked)
```

Comment:
is this sstore required?

Reply:
hm, not strictly, but it gives a quick hint on what's wrong, if the test fails (it even happened today, as I got the gas_limit wrong, I knew immediately the entire tx failed). And with `Storage` is pretty lightweight, I'd keep it in.